config/config.go

```diff
@@ -958,7 +958,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		TimeoutPrevoteDelta:         500 * time.Millisecond,
 		TimeoutPrecommit:            1000 * time.Millisecond,
 		TimeoutPrecommitDelta:       500 * time.Millisecond,
-		TimeoutCommit:               1000 * time.Millisecond,
+		TargetRoundDuration:         3500 * time.Millisecond,
 		SkipTimeoutCommit:           false,
 		CreateEmptyBlocks:           true,
```

Comment:
[for reviewers] noting that `CreateEmptyBlocks=true` is already the case and this seems desirable if we want consistent block times per https://docs.tendermint.com/v0.34/tendermint-core/configuration.html#create-empty-blocks-true